hail/src/main/scala/is/hail/expr/ir/ExtractIntervalFilters.scala

```diff
@@ -38,21 +38,32 @@ object ExtractIntervalFilters {
     }
   }
 
-  def minimumValueByType(t: Type): IntervalEndpoint = {
+  def minimumValueByType(t: Type): Any = {
     t match {
-      case _: TInt32 => endpoint(Int.MinValue, -1)
-      case _: TInt64 => endpoint(Long.MinValue, -1)
-      case _: TFloat32 => endpoint(Float.NegativeInfinity, -1)
-      case _: TFloat64 => endpoint(Double.PositiveInfinity, -1)
+      case _: TInt32 => Int.MinValue
+      case _: TInt64 => Long.MinValue
+      case _: TFloat32 => Float.NegativeInfinity
+      case _: TFloat64 => Double.PositiveInfinity
+      case _: TBoolean => false
+      case t: TLocus =>
+        val rg = t.rg.asInstanceOf[ReferenceGenome]
+        Locus(rg.contigs(0), 1)
```

Comment:
Why not `head`?

Reply:
🤷‍♂ 

Will fix so it's consistent with the other one.